### DISCUSSION TITLE
Is it possible to run AquaLinkD on Pi without a Jandy Control panel?

### DISCUSSION THREAD

**Discussion by eglass90** [aqualinkd/AqualinkD#306] [Q&A]
I have the Jandy IQpump01 interface controlling the Epump and TrueClear SWG unit. I want to run the PI with AqualinkD to control and view everything remotely. Also, can GPIO pins on the PI control relays, such as my pool Heater, since I don't have a Control panel?
Thanks, and I appreciate the response!

**Comment by sfeakes** [comment]
AqualinkD has to have a Jandy control panel for it to connect to.
